Bugsnag/Assets/Bugsnag/Editor/BugsnagEditPerformanceAssemblyDeff.cs

```diff
@@ -0,0 +1,72 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+namespace BugsnagUnity.Editor
+{
+[InitializeOnLoad]
+public class BugsnagEditPerformanceAssemblyDeff
+{
+
+    // This script fixes an issue in versions 1.7.0 and bellow of the Bugsnag Performance package.
+    // The issue is that the BugsnagPerformance.asmdef file is not correctly configured to reference the BugsnagUnity package.
+    // And therefore cannot access the version of the BugsnagUnityWebRequest class packaged with the notifier.
+    // A fix for this issue was released in the 1.7.1 version of the Bugsnag Performance package. 
+    // So this fix is only to support using legacy versions of the perf sdk.
+
+    static BugsnagEditPerformanceAssemblyDeff()
+    {
+        FixBugsnagPerformanceAsmdef();
+    }
+    public static void FixBugsnagPerformanceAsmdef()
+    {
+        string projectRoot = Directory.GetParent(Application.dataPath).FullName;
+        string packageCachePath = Path.Combine(projectRoot, "Library", "PackageCache");
+        if (!Directory.Exists(packageCachePath))
+        {
+            // Package cache directory doesn't exist, so we can't fix the asmdef
```

Comment:
Can we do any logging? Just wondered if you could suggest people add the reference themselves?